nms/flow-typed/npm/@testing-library/react_v10.x.x.js

```diff
@@ -1,19 +1,23 @@
-// flow-typed signature: 38ccc5bbcbb54d21f37adca5cc3ed7a1
-// flow-typed version: 0b0eb4afe9/@testing-library/react_v9.x.x/flow_>=v0.104.x
+// flow-typed signature: e9406c592c1d0b6096088adad78582d2
```

Comment:
No that is absolutely fine. We are in the process of [migrating the NMS to TypeScript](https://github.com/orgs/magma/projects/14) so this will anyhow soon be obsolete.

Reply:
Great! :)
The Github project link is 404 for me, JFYI :)

Reply:
The link should work now

Reply:
👍 the link works now, thanks